monkestation/code/game/objects/items/effects/jaundice.dm

```diff
@@ -0,0 +1,13 @@
+/obj/item/effect_granter/jaundice
+	name = "Jaundice Yourself"
+	icon_state = "jaundice"
+
+/obj/item/effect_granter/jaundice/grant_effect(mob/living/carbon/granter)
+	if(!HAS_TRAIT(granter, TRAIT_USES_SKINTONES))
+		to_chat(granter, span_notice("Sorry but your skin can not visibly turn jaundiced!"))
```

Comment:
maybe make the text easier to notice / bigger, and also include the fact it was refunded?